Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -42,7 +43,7 @@ void fetchInstructionsMode::execute(attitudeManager* attitudeMgr)
 
     //Note: GetFromTeleop and GetFromPM should leave their corresponding instructions unchanged and return false when they fail
```

Comment:
Is this comment relevant any more?

Reply:
It doesn't seem like it is. 
Worth considering what to do if we have failed data though...